lib/ethon/easy/http/actionable.rb

```diff
@@ -87,11 +92,23 @@ def setup(easy)
         # @param [ Easy ] easy The easy to setup.
         def set_params(easy)
           params.escape = true
+          params.rack_arrays = true if array_encoding == :rack
```

Comment:
could you add `array_encoding` to params instead? That way you wouldn't have to to the checks for `== :rack` everywhere and instead have it in one place.